subworkflows/local/samplesheet_to_channel/main.nf

```diff
@@ -126,8 +127,8 @@ workflow  SAMPLESHEET_TO_CHANNEL{
 
     // Fails when wrongfull extension for intervals file
     if (params.wes && !params.step == 'annotate') {
-    if (params.intervals && !params.intervals.endsWith("bed"))  error("Target file specified with `--intervals` must be in BED format for targeted data")
-    else log.warn("Intervals file was provided without parameter `--wes`: Pipeline will assume this is Whole-Genome-Sequencing data.")
+        if (params.intervals && !params.intervals.endsWith("bed"))  error("Target file specified with `--intervals` must be in BED format for targeted data")
```

Comment:
why the indentation?

Reply:
for better reading, it is two if cases nested

Reply:
oooh yeah, I didn't notice that